### ISSUE TITLE
Galaxy Note 9 - lock screen breaks when Magisk is installed

### ISSUE THREAD

**Issue by Xorkle** [topjohnwu/Magisk#1344]
Hi!

This is likely not Magisk's fault but seems to be triggered by its installation, possibly MagiskHide needs adjusting but I've searched everywhere and I seem to be an outlier!

I am using Magisk 19. Root works and SN passes just... kinda would like a lockscreen :D

The TL;DR is if I install Magisk, the screen lock password fails to encrypt (based on the logs I read, provided below), but the db gets populated anyway, in a broken state(?), and after rebooting the screenlock activity(?) continually goes blank and opens again (crashlooping I guess), eventually causing a reboot. Trying to boot again yields the same sadly, the issue is "fixed" by using TWRP to delete `/data/system/locksettings.db*`

On my Note9, I do the usual:

1. Format data
2. Flash https://downloadmirror.co/Wn3/no-verity-opt-encrypt-6.0.zip?download_token=ee72cdac6c6ca6082ad8a36d3812815fdc1c856c06c5af36a01c30eb472f07f3 using TWRP
3. Flash Magisk
4. Reboot into system

The boot takes about 4-5 minutes.

Once the boot is done I go through the steps of:

1. Setup, skipping everything including Samsung account sign-in, screenlock setup, and Google account login (just so I get to the debug stage quickly!)
2. I enable adb 
3. I then: `adb shell`, `su`, and `logcat`
4. And finally I turn on any screen lock, in the below log I enable a password lock:

https://files.zora.ws/log.txt
(7MB logcat, I wasn't sure how to minify it or if I should, sorry...)

Output of `select * from locksetting` on the locksetting.db retrieved from TWRP:

```
2|migrated|0|true
3|lock_screen_owner_info_enabled|0|0
4|migrated_user_specific|0|true
5|lockscreen.password_type_alternate|0|0
6|migrated_biometric_weak|0|true
7|migrated_lockscreen_disabled|0|true
8|migrated_frp|0|1
9|lockscreen.enabledtrustagents|0|com.samsung.android.service.aircommand/.remotespen.unlock.RemoteSpenTrustAgentService
15|lockscreen.samsung_biometric|0|0
18|lockout_backuppassword_attempt_deadline|0|0
19|lockout_backuppassword_attempt_timeoutms|0|0
20|lockscreen.lockoutattemptdeadline|0|0
21|lockscreen.lockoutattempttimeoutmss|0|0
24|lockscreen.profilechallenge|0|1
25|lockscreen.password_hint|0|
26|lockscreen.disabled|0|0
27|lockscreen.password_type|0|262144
28|sp-handle|0|2776151591274618949
29|lockscreen.passwordhistory|0|
42|lockscreen.failed_attempts|0|0
43|lockscreen.lockout_biometric_attempt_deadline|0|0
44|lockscreen.lockout_biometric_attempt_timeoutms|0|0
45|lockscreen.failed_fmm_attempts|0|0
```

From the logs it appears some Samsung service doesn't want the key or whatever made.

Sorry if this isn't Magisk related, but I'm at my wits end! Would be great if you have any ideas at all.

Let me know if you need further details.

**Comment by topjohnwu** [comment]
You are nuking data encryption by using the zip. It is nothing that should be asked me for, please ask the dev created the zip :)

**Comment by Xorkle** [comment]
Hi @topjohnwu,

I get the same issue if I don't flash the no-opt-verify zip, i.e. just Magisk, I also tried again with a different kernel + Magisk (in that order), same results. :(

I tried from the start, locked bootloader. Unlocked it again, made sure I don't mount /system, formatted data, flashed Magisk, rebooted (boot took 5 minutes), completed initial setup (added Google account, set passcode, tried to sign into Samsung account it said "Processing failed"), and now I can "unlock" the phone by just swiping up and the passcode setting is broken.

**Comment by Xorkle** [comment]
To clarify, only the initial boot takes 5 mins

**Comment by AndreMoraes35** [comment]
which causes lockscrem error and twrp, has nothing with the magisk, install the official rom puts only the twrp is going to continue with error in the lockscreen.  sorry the english google translate

**Comment by 4ndrii** [comment]
I experienced the same problem with my realme 5i smartphone, I tried to install the latest Magisk via the OrangeFox recovery because I had never rooted my smartphone before. but I have installed pixel 5 archipelago project. In this case, after installing magisk and successfully entering the lockscreen, here the problem occurs because I can't enter the lockscreen PIN and can't do anything, I can't even turn off my phone. But here I found a solution, namely using a google account and trying to wipe data via the Google Play application, namely find my devices and then delete it there and it worked for me
